src/components/Mission.component.jsx

```diff
@@ -15,6 +15,7 @@ function Mission({ mission }) {
 
   const joinHandler = (e) => {
     const { id } = e.target;
+    console.log(id);
```

Comment:
According JS best practice, kindly remove all console.log in your code to make it clean